src/workflows/airqo_etl_utils/satellite_utils.py

```
@@ -266,8 +281,8 @@ def process_netcdf(zip_file_path: str, variable_short_name: str) -> pd.DataFrame
                 # ds = ds.squeeze(drop=True)  # Removes dimensions of size 1 (e.g., if height dim becomes 1)
                 logger.info(f"Dataset after resampling and squeezing: {ds.dims}")
 
-                # Convert units from kg/m³ to µg/m³ (assuming this is always needed)
-                # 1 kg = 1e9 µg
+                # Convert units from kg/mÃ‚Â³ to Ã‚Âµg/mÃ‚Â³ (assuming this is always needed)
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `‚` (SINGLE LOW-9 QUOTATION MARK). Did you mean `,` (COMMA)?

(RUF003)